blog/_posts/2016-11-01-background-info.md

```diff
@@ -179,7 +179,7 @@ Which tiles just fine:
 
 I can override any other drawing function, like `rect()` or `line()`, which meant that it was time to play!
 
-I came up with a few designs, which you can view [here](https://github.com/KevinWorkman/HappyCoding/tree/gh-pages/images/backgrounds), or just look at the background of the page you're looking at right now! (Refresh to get a new random background.) You can also view the source code that generated that image by scrolling to the bottom and finding it in the footer, or by going [here](https://github.com/KevinWorkman/HappyCoding/tree/gh-pages/images/backgrounds/code).
+I came up with a few designs, which you can view [GitHub - Happy Coding - images-bg](https://github.com/KevinWorkman/HappyCoding/tree/gh-pages/images/backgrounds), or just look at the background of the page you're looking at right now! (Refresh to get a new random background.) You can also view the source code that generated that image by scrolling to the bottom and finding it in the footer, or by going [GitHub - Happy Coding - images-bg-code](https://github.com/KevinWorkman/HappyCoding/tree/gh-pages/images/backgrounds/code).
```

Comment:
Similar to my comment above, I'd prefer to avoid this kind of hyphenated link. This one could be:

`I came up with a few designed, which you can [view on GitHub]((https://github.com/KevinWorkman/HappyCoding/tree/gh-pages/images/backgrounds)`